bot/constants.py

```diff
@@ -190,12 +190,14 @@ class Emojis:
     dice_5 = "<:dice_5:755891608091885627>"
     dice_6 = "<:dice_6:755891607680843838>"
 
-    issue = "<:IssueOpen:629695470327037963>"
-    issue_closed = "<:IssueClosed:629695470570307614>"
-    pull_request = "<:PROpen:629695470175780875>"
-    pull_request_closed = "<:PRClosed:629695470519713818>"
-    pull_request_draft = "<:PRDraft:829755345425399848>"
-    merge = "<:PRMerged:629695470570176522>"
+    # These icons are from Github's repo https://github.com/primer/octicons/
+    issue_opened = "<:IssueOpen:852294683731165184>"
```

Comment:
```suggestion
    issue_open = "<:IssueOpen:852294683731165184>"
```
I think open fits a bit more to describe the issue's state, it'll also be consistent with the PR open name

Reply:
Agreed, added in https://github.com/python-discord/sir-lancebot/pull/774/commits/07a8f19be4f1178cca26330d68c778717d267415